kotsadm/web/src/components/apps/DashboardCard.jsx

```diff
@@ -212,16 +214,18 @@ export default class DashboardCard extends React.Component {
               <p className="u-fontSize--small u-color--dustyGray u-marginTop--15 u-lineHeight--medium"> Contact your account rep to update your License. </p>
             </div>
             :
-            <div>
-              <p className="u-fontSize--normal u-color--dustyGray u-marginTop--15 u-lineHeight--more"> License data is not available on this application because it was installed via Helm </p>
-            </div>
+            getingAppLicenseErrMsg ?
```

Comment:
i think this should be before `size(appLicense) > 0`, since that's what renders the license data. if this is false here, the `License data is not available on this application because it was installed via Helm` will be rendered? that doesn't make any sense.

Reply:
I'm not sure I follow. That seems like what we would want to show. If it's false there was either an error, or if not then we assume it comes from a helm chart and display that text